cpp/tests/reductions/reduction_tests.cpp

```diff
@@ -1803,8 +1916,8 @@ TYPED_TEST(DictionaryReductionTest, DISABLED_VarStd)
 
   double var   = calc_var(v, v.size());
   double std   = std::sqrt(var);
-  auto var_agg = cudf::make_variance_aggregation(/*ddof =*/1);
-  auto std_agg = cudf::make_std_aggregation(/*ddof =*/1);
+  auto var_agg = cudf::make_variance_aggregation<reduce_aggregation>(/*ddof =*/1);
+  auto std_agg = cudf::make_std_aggregation<reduce_aggregation>(/*ddof =*/1);
```

Comment:
Use a variable `auto const ddof = 1;` and refactor `calc_var` to accept `ddof` as a parameter.